fabric-command-api-v2/src/client/java/net/fabricmc/fabric/mixin/command/client/ClientPlayNetworkHandlerMixin.java

```diff
@@ -41,6 +44,14 @@ abstract class ClientPlayNetworkHandlerMixin {
 	@Final
 	private ClientCommandSource commandSource;
 
+	@Inject(method = "onGameJoin", at = @At("HEAD"))
```

Comment:
This causes race conditions as it occurs on both the netty thread and the main thread